Portal_Perfomance_Employes/Controllers/EmployeeController.cs

```diff
@@ -27,8 +26,8 @@ public async Task<IActionResult> GetEmployees()
         public async Task<IActionResult> GetEmployee(int id)
         {
             var emp = await _context.Employees
-                .FirstOrDefaultAsync(e => e.Id == id);
-            return emp == null ? NotFound("Employee with that ID was not found") : Ok(emp) ;
+                .FirstOrDefaultAsync(e => e.EmployeeId == id);
```

Comment:
Is Emloyee.Id changed to Employee.EmployeeId

Reply:
why?

Reply:
I have changed it because i created the entity USER and to be the code more clear i thought that make sense put the name of the entity together with id if necessary any join it will be more clear